cmd/helpMessages.go

```diff
@@ -365,7 +365,7 @@ Remove specified version ids of a blob from Azure Storage. Ensure that source is
 
   - azcopy rm "https://[srcaccount].blob.core.windows.net/[containername]/[blobname]" "/path/to/dir" --list-of-versions="/path/to/dir/[versionidsfile]"
 
-Remove specific blobs and virtual directories by putting their relative paths (NOT URL-encoded) in a file:
+Remove specific blobs and virtual directories by putting their relative paths (NOT URL-encoded) in a text file (i.e., list.txt) using the --list-of-files flag. In the text file, each blob and virtual directory is written on a separate line, see file contents below.
```

Comment:
From https://github.com/Azure/azure-storage-azcopy/wiki/Listing-specific-files-to-transfer:

>> The reason that it's undocumented is that, at high file counts, it has slow performance for downloads and service-to-service copies. We want to fix that before we document it more officially

We should think about whether we want to peel this text back as the param --list-of-files is marked hidden for copy commands

Reply:
list-of-files is marked hidden for copy and this particular text is for `remove`

Reply:
list-of-files incurs a performance and monetary cost, even in remove. It should be re-implemented at some point to blindly perform the operations, imo. The theory behind list-of-files is that the enumeration work is done for us, so, that would make sense to do that way.

Either way-- I'd note that here (and everywhere list-of-files is visible), if we're revealing it. I'm not sure if this same thing applies to list-of-versions, too.